backend/handlers/handlers.go

```diff
@@ -767,7 +775,12 @@ type coinFormattedAmount struct {
 }
 
 // getCoinsTotalBalance returns the total balances grouped by coins.
-func (handlers *Handlers) getCoinsTotalBalance(_ *http.Request) (interface{}, error) {
+func (handlers *Handlers) getCoinsTotalBalance(_ *http.Request) interface{} {
+	type result struct {
+		Success           bool                  `json:"success"`
+		CoinsTotalBalance []coinFormattedAmount `json:"coinsTotalBalance"`
+		ErrorCode         string                `json:"errorCode,omitempty"`
+	}
```

Comment:
same as above

Reply:
Removed the errocode in the api as discussed